.gitignore

```diff
@@ -20,4 +20,6 @@ tmp
 *.o
 *.a
 mkmf.log
-gemfiles/*.lock
\ No newline at end of file
+gemfiles/*.lock
+*.ruby-*
```

Comment:
I agree that having ruby-version and ruby-gemset in the project is good, but they were not in there already so I was sticking with what was in there.  Let me know if I should change this.